_opensearch/search-backpressure.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: default
+title: Search backpressure
+nav_order: 63
+has_children: false
+---
+
+# Search backpressure
+
+Search backpressure is a mechanism to identify resource-intensive search requests and cancel them when the node is under duress. If a search request on a node or shard has breached the resource limits and does not recover within a certain threshold, it is rejected. These thresholds are dynamic and configurable through [cluster settings](#search-backpressure-settings). 
+
+## Measuring resource consumption
+
+To decide whether to apply search backpressure, OpenSearch periodically measures the following resource consumption statistics for each search request:
+
+- CPU usage
+- Heap usage
+- Elapsed time 
+
+An observer thread periodically measures the resource usage of the node. If the node is determined to be under duress, OpenSearch examines the resource usage of each search shard task and compares it against configurable thresholds. OpenSearch considers CPU usage, heap usage and elapsed time and assigns each task a cancellation score that is then used to cancel the most resource-intensive tasks.
+
+OpenSearch limits the number of cancellations as a fraction of successful task completions and cancellations per unit time. It continues to monitor and cancel tasks until the node is no longer under duress.
+
+## Canceled queries
+
+If a query is canceled, OpenSearch may return partial results in case some shards failed. If all shards failed, OpenSearch returns an error from the server similar to the error below:
```

Comment:
To clarify the second part of this sentence, I bolded my suggestion: "If a query is canceled, OpenSearch may return partial results **if some shards failed**."